### PR TITLE
react - react1 - week2

### PR DESCRIPTION
**Pull request by VioMNE**

### CODE REVIEW

File: react/react1/week2/todo-list/src/components/ToDoItem.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export const ToDoItem = (props) => {
+    function changeTaskStatus(e) {
+      if (e.target.checked === true) {
+        e.target.previousElementSibling.className = "done";
+      } else {
+        e.target.previousElementSibling.className = "notDone";
+      }
+    }
+
```
Comment (islam-fawzy25) [inline react/react1/week2/todo-list/src/components/ToDoItem.js:9]: const [toDoStatus, setToDoStatus]=useState(false)

File: react/react1/week2/todo-list/src/components/ToDoItem.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      if (e.target.checked === true) {
+        e.target.previousElementSibling.className = "done";
+      } else {
+        e.target.previousElementSibling.className = "notDone";
+      }
+    }
+  
+    return (
+      <li className="toDoItem" key={props.todo.id} id={props.todo.id}>
+        <label className="notDone">{props.todo.description}</label>
```
Comment (islam-fawzy25) [inline react/react1/week2/todo-list/src/components/ToDoItem.js:12]: <label className={toDoStatus?"done":"notDone"}>{props.todo.description}</label>
And You need to create separate CSS file with these two class names 
.done{
    text-decoration: line-through;
}

.notDone{

}

File: react/react1/week2/todo-list/src/components/ToDoItem.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        <input type="checkbox" onChange={changeTaskStatus}></input>
```
Comment (islam-fawzy25) [inline react/react1/week2/todo-list/src/components/ToDoItem.js:13]: <input type="checkbox" onChange={()=>{setToDoStatus(!toDoStatus)}}></input>

File: react/react1/week2/todo-list/.gitignore
[the remark this answers is not in the document]
Comment (VioMNE) [reply react/react1/week2/todo-list/.gitignore:1]: HI @islam-fawzy25 , thank you so much for your feedback i will apply changes1 your feedback is very much appreciated :)

[the remark this answers is not in the document]
Comment (VioMNE) [reply react/react1/week2/todo-list/.gitignore:1]: I made changes as you suggested :)  I hope that is well :)

**Pull request merged**